autoload/phpactor/quickfix.vim

```diff
@@ -0,0 +1,166 @@
+function! phpactor#quickfix#strategy() abort
+    if !has_key(g:, 'PhpactorQuickfixStrategy')
+        return s:auto_detect_strategy()
+    endif
```

Comment:
> Would still prefer to define all available keys in the same place, which is [here](https://github.com/phpactor/phpactor/blob/develop/plugin/phpactor.vim#L34) currently. Note also that options are `camelCase`.

Can't use camel case with Funcref, it must be pascal case, otherwise vim raise an error.

> Also still not _sure_ `quickfix` is the right term.

From the vim doctumentation:
> In Vim the quickfix commands are used more generally to find a list of positions in files.

The quickfix is just the way vim show positions in file in a way that the user can jump to those positions.

Reply:
> The quickfix is just the way vim show positions in file in a way that the user can jump to those positions.

It's the way `vim` shows positions - Phpactor could use `quickfix` or not, it's an implementation for showing and jumping to positions. We could use FZF without using quickfix at all, but it's fine :)

Reply:
> We could use FZF without using quickfix at all, but it's fine, it makes sense

Absolutely, I just think we don't need a new word since we are naming a configuration variable for the vim plugin it's easier for the users to have the same language than vim. :)